.eslintrc.js

```diff
@@ -27,10 +27,11 @@ module.exports = {
     "no-shadow": "warn",
     "no-unused-vars": "off", // disabled in favour of @typescript-eslint/no-unused-vars, see https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md
     "prefer-const": "warn",
-    "radix": ["warn", "always"],
+    radix: ["warn", "always"],
     "spaced-comment": ["warn", "always", { line: { markers: ["/ <reference"] } }],
     "import/no-cycle": "warn",
     "simple-import-sort/sort": "warn",
+    "@typescript-eslint/array-type": ["warn", { default: "array-simple" }],
```

Comment:
This seems like the most confusing configuration, but I'm OK with it.

Reply:
I don't have a strong opinion here. This was used in the React projects if I recall correctly and worked well.